packages/cfd/src/Stores/Modules/index.ts

```diff
@@ -1,7 +1,10 @@
 import CFDStore from './CFD/cfd-store';
+import type { TCoreStores } from '@deriv/stores/types';
 
 export default class ModulesStore {
-    constructor(root_store) {
+    cfd;
```

Comment:
Why do we have cfd here ?

Reply:
cfd is part of Modules store {module:{cfd}}
Thats why